src/Compilers/CSharp/Portable/Symbols/Source/SourceModuleSymbol.cs

```diff
@@ -533,24 +521,6 @@ internal override void DecodeWellKnownAttribute(ref DecodeWellKnownAttributeArgu
             }
         }
 
-        private bool EmitNullablePublicOnlyAttribute
-        {
-            get
-            {
-                var compilation = DeclaringCompilation;
-                return compilation.EmitNullablePublicOnly &&
-                    compilation.IsFeatureEnabled(MessageID.IDS_FeatureNullableReferenceTypes);
```

Comment:
We used to check the feature flag, but I didn't see whether that is still handled somehow. Was that unnecessary?

Reply:
We only emit `NullablePublicOnlyAttribute` if we're also emitting `NullableAttribute` or `NullableContextAttribute`.

---
In reply to: [301355808](https://github.com/dotnet/roslyn/pull/37019#discussion_r301355808) [](ancestors = 301355808)